### PR TITLE
FTA - ET Case roles

### PR DESCRIPTION
**Pull request by SivaKanukolluQA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSAM-260

### Change description ###
Adding FTAs for case roles.

**Does this PR introduce a breaking change?** (check one with "x")
[x] No

### CODE REVIEW

File: src/functionalTest/resources/features/F-018/F-018.feature [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    Given a user with [an active IDAM profile with full permissions],
+    And a user [Befta1 - who is the actor for requested role],
+    And a successful call [to create org role assignments for actors & requester] as in [S-269_Org_Role_Creation],
+    And a successful call [to create role assignments for requested role] as in [S-269_Access_Requested],
+    When a request is prepared with appropriate values,
+    And the request [contains specific-access-judiciary case granted role assignment],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a positive response is received,
+    And the response has all other details as expected.
+    And a successful call [to delete role assignments just created above] as in [S-269_DeleteDataForRoleAssignmentsForOrgRoles].
```
Comment (mattnayler) [inline src/functionalTest/resources/features/F-018/F-018.feature:123]: Are we missing the second delete from here: i.e. `S-269_DeleteDataForRoleAssignmentsForGrantedRole`?

Comment (SivaKanukolluQA) [reply src/functionalTest/resources/features/F-018/F-018.feature]: Removed the S-269 test as we are not able to remove the specific access case until it's expiry.

**Review by mattnayler — APPROVED** [review]
LGTM

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Pull request merged**